components/filters.tsx

```
@@ -44,77 +78,85 @@ function BlockLink({
   categories,
-  total,
 }: {
-  total: number;
   categories: Category[];
 }) {
   const t = useTranslations("listing");
   const pathname = usePathname();
 
+  const totalItems = categories.reduce((sum, cat) => sum + (cat.count || 0), 0);
+
+  const truncateText = (text: string, maxLength: number = 20) => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)